gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "10.0.0",
+  "version": "10.1.0",
```

Comment:
Should be a patch version

Reply:
> Should be a patch version

Updated here: 8401f44a8e328618e7a323886e36bbd9fd25e71c